app/controllers/topics.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+  newTopic: Ember.computed(function () {
+    return this.store.createRecord('topic');
+  }),
+
+  replaceNewTopic () {
+    this.set('newTopic', this.store.createRecord('topic'));
```

Comment:
Almost feels like you should leverage `newTopic` in some way, as `this.store.createRecord('topic')` is reused.